htdocs/product/stock/class/mouvementstock.class.php

```diff
@@ -660,9 +660,16 @@ function get_origin($fk_origin, $origintype)
 				require_once DOL_DOCUMENT_ROOT.'/fourn/class/fournisseur.facture.class.php';
 				$origin = new FactureFournisseur($this->db);
 				break;
-
+				
 			default:
-				return '';
+				if ($origintype)
+				{
+					require_once DOL_DOCUMENT_ROOT.'/'.$origintype.'/class/'.$origintype.'.class.php';
```

Comment:
I think it is better to use dol_include_once instead of requiere once  for compatibility with external modules if custom directory is enabled